src/soulmods/src/games/x64/eldenring.rs

```diff
@@ -18,14 +18,34 @@ use ilhook::x64::{Hooker, HookType, Registers, CallbackOption, HookFlags, HookPo
 use mem_rs::prelude::*;
 use log::info;
 use crate::util::GLOBAL_VERSION;
+use crate::util::Version;
+
+struct FpsOffsets
+{
+    target_frame_delta: isize,
+    frame_delta: isize,
+    timestamp_previous: isize,
+    timestamp_current: isize,
+    timestamp_history: isize,
```

Comment:
maybe these should be usize

Reply:
`pointer.offset()` wants `isize`. Alternatively, we could use `pointer.add()` if you want to use `usize` instead?